website/docs/r/rds_instance_v3.html.markdown

```diff
@@ -153,9 +153,9 @@ The following arguments are supported:
   Specifies the security group which the RDS DB instance belongs to.
   Changing this parameter will create a new resource.
 
-* `subnet_id` -
+* `network_id` -
```

Comment:
As pointed out by @niuzhenguo there should be a smooth migration.
https://www.terraform.io/docs/extend/best-practices/deprecations.html#renaming-a-required-attribute

* `subnet_id` 
**Deprecated**
(Optional)
Specifies the subnet id. Changing this parameter will create a new resource. Either subnet id or network id MUST be specified.
* `network_id ` 
(Optional)
Specifies the network id. Changing this parameter will create a new resource. Either subnet id or network id MUST be specified.

Reply:
@r0b2g1t Why is this conversation marked as resolved?